app/models/volunteer_op_form.rb

```diff
@@ -8,13 +8,19 @@ class DoitVolunteerOp
     validates :advertise_end_date, presence: true
     validates :doit_org_id, presence: true
     attr_accessor :advertise_start_date, :advertise_end_date, :doit_org_id
-
   end
 
-  attr_accessor :volunteer_op, :post_to_doit, :doit_volunteer_op
+  attr_writer :volunteer_op, :post_to_doit, :doit_volunteer_op
```

Comment:
maybe we can actually delete the writer? maybe not ... something for a refactoring ticket later I guess ...